src/encoder.rs

```diff
@@ -2155,12 +2174,33 @@ fn encode_tile(sequence: &mut Sequence, fi: &FrameInvariants, fs: &mut FrameStat
             for i in 0..INTER_REFS_PER_FRAME {
                 let r = fi.ref_frames[i] as usize;
                 if pmvs[r].is_none() {
-                    pmvs[r] = estimate_motion_ss4(fi, fs, r, &bo);
+                    pmvs[r] = frame_pmvs[sby * fi.sb_width + sbx][r];
                     if let Some(pmv) = pmvs[r] {
-                        pmvs[r + 1*REF_FRAMES] = estimate_motion_ss2(fi, fs, r, &sbo.block_offset(0, 0), &pmv);
-                        pmvs[r + 2*REF_FRAMES] = estimate_motion_ss2(fi, fs, r, &sbo.block_offset(8, 0), &pmv);
-                        pmvs[r + 3*REF_FRAMES] = estimate_motion_ss2(fi, fs, r, &sbo.block_offset(0, 8), &pmv);
-                        pmvs[r + 4*REF_FRAMES] = estimate_motion_ss2(fi, fs, r, &sbo.block_offset(8, 8), &pmv);
+                        let pmv_w = if sbx > 0 {
```

Comment:
I guess this implements the #594? i.e. Referencing neighboring SB's MVs. That is great!